CI/test_coverage.py

```diff
@@ -41,7 +41,7 @@ def call(cmd):
 
 ret, gcovr_version_text = check_output(["gcovr", "--version"])
 gcovr_version = tuple(
-    map(int, re.match("gcovr (\d+\.\d+)", gcovr_version_text).group(1).split("."))
+    map(int, re.match(r"gcovr (\d+\.\d+)", gcovr_version_text).group(1).split("."))
```

Comment:
@paulgessinger same comment here with the `\.`

Reply:
Same here. With the raw string we better keep it escaped.

Reply:
If it's not escaped it will match any character. `\.` matches the dot character literally.